esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -886,6 +893,11 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
     else
         ESPLOG(LogMin,"DESDL: Transaction ID could not be generated!");
 
+    // In the future we could instantiate a profile from the service configuration.
+    // For now use a profile created on service initialization.
+    //context.setTxSummaryProfile(m_txSummaryProfile.get());
+    context.setTxSummaryProfile(m_txSummaryProfile);
```

Comment:
This is nitpicking over a personal preference, but I would like to see this done before adding values. It doesn't matter either way now, but if there is ever a change where the profile is used sooner than expected today I would like to have the profile in place as soon as possible.

Ideally the profile would be made available for the construction of the summary, but that might not be warranted yet.

Reply:
Ok that makes sense. I'll add that task to the PR for next batch of changes.